rpcserver.go

```diff
@@ -1813,6 +1815,51 @@ func GetUnconfirmedBalance(w *Wallet, chainSvr *chain.Client, icmd btcjson.Cmd)
 	return (unconfirmed - confirmed).ToUnit(btcutil.AmountBTC), nil
 }
 
+// ImportAddress handles an importaddress request by parsing
+// a encoded address and importing it as a watch-only address to the
+// imported address account.
+func ImportAddress(w *Wallet, chainSvr *chain.Client, icmd btcjson.Cmd) (interface{}, error) {
+	cmd := icmd.(*btcjson.ImportAddressCmd)
+
+	addr, err := btcutil.DecodeAddress(cmd.Address, activeNet.Params)
+	if err != nil {
+		return nil, btcjson.ErrInvalidAddressOrKey
```

Comment:
You can use the error code from this var, but please add extra context in the error message with the error string.